rpm/Makefile

```diff
@@ -101,28 +94,16 @@ rpmbuild/SOURCES/cli.tgz:
 		alpine \
 		tar -C / -c -z -f /v/cli.tgz --exclude .git cli
 
-rpmbuild/SOURCES/containerd-proxy.tgz:
-	mkdir -p tmp/
-	curl -fL -o tmp/containerd-proxy.tgz "https://github.com/crosbymichael/containerd-proxy/archive/$(CONTAINERD_PROXY_COMMIT).tar.gz"
-	tar xzf tmp/containerd-proxy.tgz -C tmp/
-	mv tmp/containerd-proxy-$(CONTAINERD_PROXY_COMMIT) tmp/containerd-proxy
-	mkdir -p $(@D)
-	$(CHOWN) -R $(shell id -u):$(shell id -g) rpmbuild
-	tar -zcf $@ -C tmp/ containerd-proxy
-	rm -rf tmp/
-
 rpmbuild/SOURCES/docker.service: ../systemd/docker.service
 	mkdir -p $(@D)
 	cp $< $@
 
-rpmbuild/SOURCES/dockerd.json: ../common/dockerd.json
+rpmbuild/SOURCES/engine-image:
+	$(MAKE) -C ../image image-linux-dm
+	mkdir -p $(@D)
+	cp ../image/image-linux-dm $@
```

Comment:
fwiw, it seems like this is kind of deep in the tree for the CI to be injecting it from the S3 bucket - I worry a tad this might increase the chance we inadvertently wind up building the image again during the package phase of the pipeline, but if you've got this wired up already, lets run with it for now.

Reply:
Already have it wired up, so there's no worry